starter-code/index.html

```diff
@@ -1,51 +1,108 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="../style/style.css">
-  <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon-32x32.png">
- <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Red+Hat+Display&display=swap" rel="stylesheet">
-  <title>Meet landing page</title>
-</head>
-<body>
-  <div class="main-container">
-    <div class="head-meet"><img src="../starter-code/assets/logo.svg" alt=""></div>
-    <div class="header">
-      <div class="left-bubbles">
-        <div class="bubble-one"></div>
-        <div class="bubble-two"></div>
-        <div class="bubble-three"></div>
-        <div class="bubble-four"></div>
-        <div class="bubble-five"></div>
-        <div class="bubble-six"></div>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../style/style.css" />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="./assets/favicon-32x32.png"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Red+Hat+Display&display=swap"
+      rel="stylesheet"
+    />
+    <title>Meet landing page</title>
+  </head>
+  <body>
+    <div class="main-container">
```

Comment:
This page should look like this:
![Screenshot from 2021-09-21 21-24-33](https://user-images.githubusercontent.com/90243506/134235114-6d71f7e4-7115-4d6c-8dc9-612e0880724f.png)